File: Cplusplus/URI/01-iniciantes/02-Nivel/1008.cpp
#include <iostream>
#include <iomanip>
using namespace std;
int main()
{   
    int idWorker , workedHours;
    double hourlywages;
    cout << "Digite o numero do funcionario" << endl;
    cin >> idWorker;
    cout << "Digite as horas trabalhadas do funcionario" << endl;
    cin >> workedHours;
    cout << "Digite o valor hora do funcionario" << endl;
    cin >> hourlywages;
    cout << "NUMBER = " << idWorker << endl;
    cout << "SALARY = U$ "<< fixed << setprecision(2) << (hourlywages * workedHours) << endl;
    return 0;
}

File: Csharp/URI/01-Iniciantes/02-Nivel/1008.cs
using System;

namespace Exe1008.URI
{
    public class Exe1008
    {
        public  Exe1008()
        {   
            int idWorker , workedHours;
            double hourlywages;
            Console.WriteLine("Digite o Id do funcionario!");
            idWorker = int.Parse(Console.ReadLine());
            Console.WriteLine("Digite as Horas trabalhadas");
            workedHours = int.Parse(Console.ReadLine());
            Console.WriteLine("Digite o valor hora");
            hourlywages = double.Parse(Console.ReadLine());
            Console.WriteLine("NUMBER = {0}",idWorker);
            Console.WriteLine("SALARY = U$ {0:0.00}", (hourlywages * workedHours));  
              
                   
        }
    }
}